dll/shellext/shellbtrfs/main.cpp

```diff
@@ -499,7 +497,7 @@ static void unreg_prop_sheet_handler(const wstring& filetype, const wstring& nam
 #endif
 }
 
-extern "C" STDAPI DllRegisterServer(void) {
+STDAPI DllRegisterServer(void) {
```

Comment:
I am pretty sure that this won't work like this anymore, did you test it?

Reply:
by definition STDAPI implies extern "C". Or did I miss something? 
and yes it's tested and working

Reply:
Ah right, it is indeed part of that already.
Without the `extern "C"` thing gcc usually fails to act normal.